### PR TITLE
feat: Lists comp added 

### PR DESCRIPTION
**Pull request by jatinmore**
-List varients added
-main content overflow fix
-screenshots
127 0 0 1_5502_components_List_list html (1)

127 0 0 1_5502_components_List_list html (2)

### CODE REVIEW

**Review by MiheerTamkhane — APPROVED** [review]
Looks good, nice work

File: components/List/list.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+
+                    <li class="aside-item">
+                        <a class="aside-link" href="/components/Image-Input/input.html">Input</a>
+                    </li>
+
+                    <li class="aside-item">
+                        <a class="aside-link active" href="/components/List/list.html">List</a>
+                    </li>
+
```
Comment (shubhmehra) [inline components/List/list.html:73]: you can format this code with prettier

File: styles/list.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import url('/styles/main.css');
+.lists {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+    border: 1px solid black;
```
Comment (shubhmehra) [inline styles/list.css:6]: use colors from root for better consistency

File: styles/list.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.stack-list {
+    border: 1px solid black;
+}
+
+.stack-item {
+    display: flex;
+    border-bottom: 1px solid black;
+    padding: 0.5rem;
```
Comment (shubhmehra) [inline styles/list.css:29]: same

**Review by shubhmehra — APPROVED** [review]
approved but small comments

**Pull request merged**